eido/validation.py

```diff
@@ -30,12 +32,15 @@ def _validate_object(object, schema, exclude_case=False):
     :param bool exclude_case: whether to exclude validated objects
         from the error. Useful when used ith large projects
     """
-    try:
-        jsonschema.validate(object, schema)
-    except jsonschema.exceptions.ValidationError as e:
-        if not exclude_case:
-            raise
-        raise jsonschema.exceptions.ValidationError(e.message)
+
+    validator = Draft7Validator(schema)
+    if validator.is_valid(object):
```

Comment:
Wondering if maybe we could just make this a single "if not valid" statement. I.e.:

```python
if not validator.is_valid(object):
    errors = sorted(validator.iter_errors(object), key=lambda e: e.path)
        raise EidoValidationError(
            f"Validation unsuccessful. {len(errors)} errors found.", errors
        )
```

Since `if validator.is_valid(object): pass` is the same as no code. It's probably fine either way, though!